plugin/trino-phoenix5/pom.xml

```diff
@@ -16,6 +16,12 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <dep.hbase.version>2.2.6</dep.hbase.version>
+
+        <!-- This is required for JDK 17 to start HBase server due to illegal reflective access -->
+        <air.test.jvm.additional-arguments>
+            --add-opens=java.base/sun.nio.ch=ALL-UNNAMED
+            --add-opens=java.base/java.nio=ALL-UNNAMED
+        </air.test.jvm.additional-arguments>
```

Comment:
Unless this can be removed with Phoenix 5.2, this probably belongs in the other commit.

Also, I'd probably put the comment _inside_ the tag in case any more flags are added here in the future (or I'd specify that HBase needs reflective access just to java.nio and sun.nio.ch in the comment).